pymc_marketing/mmm/tvp.py

```diff
@@ -92,7 +92,40 @@
 from pymc.distributions.shape_utils import Dims
 
 from pymc_marketing.constants import DAYS_IN_YEAR
-from pymc_marketing.hsgp_kwargs import HSGPKwargs
+from pymc_marketing.hsgp_kwargs import HSGP, CovFunc, HSGPKwargs
```

Comment:
Why not to move `tvp` into components folder? Maybe with a better name. I'm thinking with HSGP we are modeling latent variables, maybe `latent.py` under components. Replacing `hsgp_kwargs` instead.

`from pymc_marketing.components.latent import HSGP, CovFunc, HSGPKwargs`

Reply:
Moved to hsgp module